src/components/Modal/LoginModal.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+function LoginModal() {
+  const { Kakao } = window;
+  const history = useHistory();
```

Comment:
```suggestion
import styled from 'styled-components';

const { Kakao } = window;

function LoginModal() {
  const history = useHistory();
```
컴포넌트 바깥에 옮겨도 좋을 것 같습니다. 굳이 렌더 될 때마다 Kakao를 재할당할 필요가 없어서요